### DISCUSSION TITLE
Question about setting Label Width for input components

### DISCUSSION THREAD

**Discussion by Dimasikkk1** [dotnetcore/BootstrapBlazor#2987] [Q&A] [answered]
I want to create a custom EditTemplate for the table. I saw that EditForm and the --bb-row-label-width style have such a property.
Is it acceptable to put an EditForm in an EditTemplate, or is it better to set the --bb-row-label-width style for input components?
When I put the EditForm in the EditTemplate, I get a `<div class="form-body">` inside the same `<div class="form-body">`. It seems to me that I did it wrong.

I'm new to Blazor and Bootstrap 👉👈

**Comment by bb-auto** [comment]
@Dimasikkk1 Thank you for submitting. We will give feedback later.

**Comment by ArgoZhang** [comment, marked as answer]
@Dimasikkk1 put the following code `app.css` in your project.

```
:root {
    --bb-row-label-width: 200px;
}
```

**Reply by Dimasikkk1** [reply]
What if I need to set this individually for different input components in the form?

**Reply by ArgoZhang** [reply]
@Dimasikkk1 use css isolation like `xx.razor.css`
